datasets/np/mha_sanctions/crawler.py

```diff
@@ -0,0 +1,81 @@
+import csv
+from typing import Dict
+import zavod.helpers as h
+from zavod import Context
+
+DATE_FORMATS = ["%m/%d/%Y"]
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    unsc_id = row.pop("code")
+    entity_type = row.pop("type")
+    name = row.pop("name")
+    # Split `alias` on `;` and trim any extra whitespace
+    alias = row.pop("alias").split(";")
+    birth_date = h.parse_date(row.pop("date of birth"), DATE_FORMATS)
+    birth_place = row.pop("place of birth").split(";")
+    nationality = row.pop("nationality").split(";")
+    pass_no = row.pop("passport no").split(";")
+    national_id = row.pop("national id")
+    address = row.pop("address").split(";")
+    listing_date = h.parse_date(row.pop("listed on"), DATE_FORMATS)
+    sanction_status = row.pop("sanction status")
+    remarks = row.pop("remarks")
+
+    entity = None
+    if entity_type == "INDIVIDUALS":
+        entity = context.make("Person")
+        entity.id = context.make_id(name, birth_date, unsc_id)
+        entity.add("name", name)
+        for a in alias:  # Add aliases
+            entity.add("alias", a.strip())
+        for d in birth_date:
+            entity.add("birthDate", d.strip())
+        for p in birth_place:
+            entity.add("birthPlace", p.strip())
+        for n in nationality:
+            entity.add("nationality", n.strip())
+        for p in pass_no:
+            entity.add("passportNumber", p.strip())
+        entity.add("idNumber", national_id)
+        for a in address:
+            entity.add("address", address)
+        entity.add("notes", remarks)
+        if sanction_status == "ACTIVE":
+            entity.add("topics", "sanction")
+            sanction = h.make_sanction(context, entity)
+            sanction.add("listingDate", listing_date)
+            sanction.add("unscId", unsc_id)
+            context.emit(entity, target=True)
+            context.emit(sanction)
+        elif sanction_status == "REMOVED":
+            context.emit(entity, target=False)
```

Comment:
it's nice to still have the sanction entity with the information about the sanction, but just not give them the sanction topic.

When customers ask how they should know why someone came up as a match in the dataset, we tell them to look primarily at the topic property. If there's no topic, the entity is usually there because they're either somehow related to some risky entity, but not officially a risk themselves, or they used to be sanctioned/debarraed, but aren't any more.

Reply:
since we're still including the person in the dataset, it's nice to include the sanction information explaining why they're here, but also why they don't have the topic any more.